components/widgets/WindspeedHourly/index.js

```diff
@@ -0,0 +1,110 @@
+import PropTypes from "prop-types";
+import { windspeedUnitType } from "@/components/shapes/units";
+import { useTranslation } from "react-i18next";
+import * as Styled from "./styles";
+import ScrollableHorizontalWrapper from "@/components/atomic/ScrollableHorizontalWrapper";
+import UniqueIconComposer from "@/components/svg/UniqueIconComposer";
+import { ScreenreaderText } from "@rubin-epo/epo-react-lib";
+
+const WindspeedHourly = ({ unit, windspeedData = [] }) => {
+  const {
+    t,
+    i18n: { language = "en" },
+  } = useTranslation();
+
+  const timeFormatter = new Intl.DateTimeFormat(language, {
+    timeStyle: "short",
+    hourCycle: "h23",
+  });
+
+  const directionFormatter = new Intl.NumberFormat(language, {
+    style: "unit",
+    unit: "degree",
+  });
+
+  const speedFormatters = {
```

Comment:
set up a custom formatter for each windspeed unit, sadly knots is not included in the list of available `units` so it has to be formatted manually using the decimal formatter